ocs_ci/utility/networksplit/main.py

```diff
@@ -0,0 +1,272 @@
+    # schedule both timers on every node of the cluster
+    for node in _list_cluster_nodes():
+        node_name = node[5:]
+        cmd_str = f"systemctl start {start_unit} {stop_unit}"
+        exec_oc_debug_cmd(node_name, cmd_str, timeout=300)
+
+
+def main_setup():
+    """
+    Simple command line interface to generate MachineConfig yaml to deploy to
+    make scheduling network splits possible.
+
+    Example usage::
+
+         $ ocs-network-split-setup --zone-label-names arbiter,d1,d2 -o mc.yaml
+         $ oc create -f mc.yaml
+         $ oc get mcp
+
+    """
+    ap = argparse.ArgumentParser(description="network split setup helper")
+    ap.add_argument(
+        "--print-env-only",
+        action="store_true",
+        default=False,
+        help="just show firewall zone env file and exit",
+    )
+    ap.add_argument(
+        "--output",
+        "-o",
+        metavar="FILE",
+        type=argparse.FileType("w"),
+        default=sys.stdout,
+        help="name of yaml file with MachineConfig to deploy on OCP cluster",
+    )
+    ap.add_argument(
+        "--zone-label-names",
+        required=True,
+        help="comma separated list of zone names 'a,b,c' where a is an arbiter",
+    )
+    ap.add_argument(
+        "-d",
+        "--debug",
+        action="store_true",
+        help="set log level to DEBUG",
+    )
+    args = ap.parse_args()
+
+    if args.debug:
+        logging.basicConfig(level=logging.DEBUG)
+
+    # get node ip addresses of each zone via zone config
+    zone_names = args.zone_label_names.split(",")
+    zone_config = get_zone_config(zone_names)
+    zone_env = zone_config.get_env_file()
+
+    if args.print_env_only:
+        print(zone_env)
+        return
+
+    # get MachineConfig spec (ready to deploy list of dics)
+    mc = get_networksplit_mc_spec(zone_env)
+    args.output.write(yaml.dump_all(mc))
+
+
+def main_schedule():
+    """
+    Simple command line interface to schedule given cluster network split.
+
+    Example usage::
+
+         $ ocs-network-split-sched ab-bc 2021-03-18T18:45 --split-len 30
+
+    """
+    ap = argparse.ArgumentParser(description="network split scheduler")
+    ap.add_argument(
+        "split_name",
+        choices=VALID_NETWORK_SPLITS,
+        help="which split configuration to schedule",
+    )
+    ap.add_argument(
+        "timestamp", help="moment when to schedule the network split (in ISO format)"
+    )
+    ap.add_argument(
+        "--split-len",
+        metavar="MIN",
+        default=15,
+        help="how long the network split should take (in minutes)",
+    )
+    ap.add_argument(
+        "-d",
+        "--debug",
+        action="store_true",
+        help="set log level to DEBUG",
+    )
+    args = ap.parse_args()
+
+    if args.debug:
+        logging.basicConfig(level=logging.DEBUG)
+    try:
+        start_dt = datetime.fromisoformat(args.timestamp)
+    except ValueError as ex:
+        print(ex)
```

Comment:
Should this be logged instead of printed?

Reply:
This is a code of command line tool, printing is fine here.